modules/client/js/src/main/scala/com/azavea/stac4s/api/client/package.scala

```diff
@@ -1,5 +1,8 @@
 package com.azavea.stac4s.api
 
 package object client {
-  type SttpStacClient[F[_]] = SttpStacClientF[F, SearchFilters]
+  type SttpStacClient[F[_]]            = SttpStacClientF[F, SearchFilters]
+  type StacClient[F[_]]                = StacClientF[F, SearchFilters]
+  type StreamingClient[F[_]]           = StreamingStacClientF[F, fs2.Stream[F, *], SearchFilters]
+  type StreamingStacClient[F[_], G[_]] = StreamingStacClientF[F, G, SearchFilters]
```

Comment:
This still should be type aliases, traits would not work since they add children into the traits hierarchy.